src/compiler/diagnosticMessages.json

```
@@ -3048,6 +3048,10 @@
         "category": "Error",
         "code": 2795
     },
+    "It is likely that you are missing a comma to separate these two template expressions. They form form a tagged template expression which is invalid.": {

```

Apart from the duplicate "form form", @sandersn what do you think?

```suggestion
    "It is likely that you are missing a comma to separate these two template expressions. They form a tagged template expression which cannot be invoked.": {
```

---

Fixed

---

Yes, that sounds reasonable to me.